centos7-viaq.repo

```diff
@@ -40,11 +40,17 @@ gpgcheck=0
 [centos-openshift-origin38-candidate]
 name=CentOS OpenShift Origin38 Candidate
 baseurl=http://cbs.centos.org/repos/paas7-openshift-origin38-candidate/x86_64/os
-enabled=0
+enabled=1
 gpgcheck=0
 
 [centos-openshift-origin39-candidate]
 name=CentOS OpenShift Origin39 Candidate
 baseurl=http://cbs.centos.org/repos/paas7-openshift-origin39-candidate/x86_64/os
+enabled=1
+gpgcheck=0
+
+[centos-openshift-origin310-candidate]
+name=CentOS OpenShift Origin310 Candidate
+baseurl=http://cbs.centos.org/repos/paas7-openshift-origin310-candidate/x86_64/os
```

Comment:
Why add version 10 if you dont supply the ansible-inventory for it?
Can users use openshift-origin 3.10?

Reply:
no - removed 3.10